templates/demo-store/tailwind.config.js

```diff
@@ -1,25 +1,16 @@
 import formsPlugin from '@tailwindcss/forms';
 import typographyPlugin from '@tailwindcss/typography';
 
-function withOpacityValue(variable) {
-  return ({opacityValue}) => {
-    if (opacityValue === undefined) {
-      return `rgb(var(${variable}))`;
-    }
-    return `rgb(var(${variable}) / ${opacityValue})`;
-  };
-}
-
 /** @type {import('tailwindcss').Config} */
 export default {
   content: ['./app/**/*.{js,ts,jsx,tsx}'],
   theme: {
     extend: {
       colors: {
-        primary: withOpacityValue('--color-primary'),
-        contrast: withOpacityValue('--color-contrast'),
-        notice: withOpacityValue('--color-accent'),
-        shopPay: 'var(--color-shop-pay)',
+        primary: 'rgba(var(--color-primary), <alpha-value>)',
+        contrast: 'rgba(var(--color-contrast), <alpha-value>)',
+        notice: 'rgba(var(--color-accent), <alpha-value>)',
+        shopPay: 'rgba(var(--color-shop-pay), <alpha-value>)',
```

Comment:
`rgba` seems to be considered legacy and it doesn't work for me in the current version of Tailwind. It works with the [modern `rgb`](https://tailwindcss.com/blog/tailwindcss-v3-1#easier-css-variable-color-configuration), though.

```suggestion
        primary: 'rgb(var(--color-primary) / <alpha-value>)',
        contrast: 'rgb(var(--color-contrast) / <alpha-value>)',
        notice: 'rgb(var(--color-accent) / <alpha-value>)',
        shopPay: 'rgb(var(--color-shop-pay) / <alpha-value>)',
```